elasticdl/python/worker/worker.py

```diff
@@ -423,9 +423,18 @@ def report_gradient_to_ps(self, grads):
         # TODO: choose the last response temporarily
         return res.accepted, res.model_version
 
-    # TODO: Reuse common code in report_gradient_to_ps and
-    # report_gradient_to_master
     def report_gradient_locally(self, grads):
+        # TODO: Automatically replace EDL embedding layer
```

Comment:
I think for AllReduce strategy, the ElasticDL embedding layer shouldn't exist. Raising exception is fine. The TODO is not necessary.

Reply:
Thanks. Removed.